ah_bootstrap.py

```diff
@@ -210,6 +212,8 @@ def main(cls, argv=None):
         auto_use = config.pop('auto_use', False)
         bootstrapper = cls(**config)
 
+        _builtins._astropy_helpers_config = config
```

Comment:
Sounds good I think.  Let me have a quick look at the setup.cfg parsing.  I don't think we actually need to write anything for that that isn't already there. 

I think also there's currently an ah_bootstrap section, but I also want to add or change that to astropy-helpers, since there are some other config values related to it that it would be nice to have in a config file.